cmd/idracsdkapps/elasticsearch/elasticsearchplugin.go

```diff
@@ -0,0 +1,336 @@
+package main

```

Comment:
Ramaraju's stuff is a great start.  We'll want to clean it up before we publish to main.  (efficiency problems, bugs, hard coded values).  Can we publish it into our private forks, clean it up before publishing it to main.  If we want to consider the story to be completed, maybe the story can track the first step?